pontoon/sync/formats/json_keyvalue.py

```diff
@@ -37,9 +37,9 @@ def __init__(self, order, key, context, source_value, value):
             key=key,
             context=context,
             source_string=source_value,
-            strings={None: value}
-            if value
-            else {},  # No plural support in key value JSON
+            strings=(
+                {None: value} if value else {}
+            ),  # No plural support in key value JSON
```

Comment:
I kept this one in, although not strictly necessary (it was done by black 24, while we use 22). The previous version doesn't seem particularly readable.